### DISCUSSION TITLE
Add a Read later feature to the extension

### DISCUSSION THREAD

**Discussion by yTi7** [dailydotdev/daily#866] [Feature Requests]
### Description:
I love using daily.dev to stay updated on the best developer news. However, sometimes I don't have enough time to read all the articles that interest me. I would like to have a way to save them for later reading without cluttering my bookmarks or losing track of them.

A possible solution is to add a Read later feature to the extension, which would allow users to mark articles as read later and access them from a dedicated tab or section. This way, users can easily find and read the articles they saved when they have more time.

### Benefits: 
This feature would enhance the user experience and satisfaction of daily.dev by providing a convenient and organized way to save and read articles later. It would also increase user engagement and retention by encouraging users to come back to daily.dev more often and read more content.

### Possible challenges: 
This feature would require some changes to the extension UI and backend, such as adding a button or icon to mark articles as read later, creating a new tab or section to display the saved articles, and storing and syncing the user's read later list across devices. It would also need a notification system to remind about the articles marked as read later.

**Comment by idoshamun** [comment]
Hi @yTi7,

That's a great suggestion. We already have it reported here https://github.com/dailydotdev/daily/discussions/364 so let's keep the discussion in one place please.
